odl_video/settings.py

```diff
@@ -480,3 +484,4 @@ def get_all_config_keys():
 ]
 # This is necessary for loading USwitch video player code from an external source
 USWITCH_URL = get_string('USWITCH_URL', '')
+LECTURE_CAPTURE_USER = get_string('LECTURE_CAPTURE_USER', 'admin')
```

Comment:
I do not think we need a default inexistent user here that might fail.

I also would like something that verify that the user set here actually exists at the app startup (where I added the settings verifications) so that we can return a meaningful error in case it does not exist.